api/reporting/schemas/latest/workspace.json

```diff
@@ -0,0 +1,143 @@
+[
```

Comment:
What is the significance of calling this directory "latest"? Does this imply that we're going to have divergent schemas across snapshots? I wouldn't think this works in BigQuery

Reply:
An open question for [RW-5195](https://precisionmedicineinitiative.atlassian.net/browse/RW-5195) is where to host these schema files and any docs for the users. Keeping them in this directory would likely work, but it feels sloppy to have them rely on something buried in our repo and not at the top level someplace. On the other hand, making a new repo just for this purpose seems like overkill.